package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "build": "babel src -d lib",
-    "test": "mocha --compilers js:babel-register ./tests/guessTheNumber_test.js"
+    "test": "mocha --compilers js:babel-register ./tests/stringIndices_test.js"
```

Comment:
Change this to ./tests/*.js. If you want to run one test at a time, put a .only on the describe in the test file: `describe.only`